src/internal/connector/onedrive/item.go

```diff
@@ -91,14 +90,14 @@ func driveItemInfo(di models.DriveItemable, itemSize int64) *details.OneDriveInf
 
 // driveItemWriter is used to initialize and return an io.Writer to upload data for the specified item
 // It does so by creating an upload session and using that URL to initialize an `itemWriter`
+// TODO: @vkamra verify if var session is the desired input
 func driveItemWriter(
 	ctx context.Context,
 	service graph.Service,
 	driveID, itemID string,
 	itemSize int64,
 ) (io.Writer, error) {
-	// TODO: @vkamra verify if var session is the desired input
-	session := msup.NewCreateUploadSessionPostRequestBody()
+	session := models.NewUsersItemDrivesItemItemsItemCreateUploadSessionPostRequestBody()
```

Comment:
> `ItemDrivesItemItemsItemCreate`

😆

Reply:
It is terrible... but the library reduction is everything.